<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskListWidgetsClass</class>
 <widget class="QWidget" name="TaskListWidgetsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addpushButton">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delpushButton">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="avs_setuppushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resource.qrc">
         <normaloff>:/new/1_1image/Resource/image/UVAIGC/gearshape.svg</normaloff>:/new/1_1image/Resource/image/UVAIGC/gearshape.svg</iconset>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="TaskQListWidget" name="listWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="Drag_ui_widget" native="true">
     <property name="styleSheet">
      <string notr="true">#Drag_ui_widget{
border: 2px dashed rgb(0, 85, 0);
border-radius:5px;

}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="Drag_gridLayout">
        <item row="1" column="1">
         <widget class="QPushButton" name="drag_pushButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Resource.qrc">
            <normaloff>:/new/prefix1/Resource/image/winOCRUI/square.and.arrow.down.svg</normaloff>:/new/prefix1/Resource/image/winOCRUI/square.and.arrow.down.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>将音频或视频文件拖放到这里</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>一款通过AI人工智能 准确分离人声和背景音乐的软件</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>使用流程:设置窗口配置模型文件--&gt;添加文件--&gt;“开始”分离</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>输出路径:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outpath_label_pushButton">
       <property name="text">
        <string>PushButton</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="out_path_pushButton">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QProgressIndicator" name="Cancel_pIndicator" native="true">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stoppushButton">
       <property name="text">
        <string>停止</string>
       </property>
       <property name="icon">
        <iconset resource="../Resource.qrc">
         <normaloff>:/new/prefix1/Resource/image/winOCRUI/pause.svg</normaloff>:/new/prefix1/Resource/image/winOCRUI/pause.svg</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runpushButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">#runpushButton222{
 color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:10px;
    height:30px;
    width:100px;

}</string>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
       <property name="icon">
        <iconset resource="../Resource.qrc">
         <normaloff>:/new/prefix1/Resource/image/winOCRUI/play.svg</normaloff>:/new/prefix1/Resource/image/winOCRUI/play.svg</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QProgressIndicator</class>
   <extends>QWidget</extends>
   <header>QProgressIndicator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TaskQListWidget</class>
   <extends>QListWidget</extends>
   <header>TaskQListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
